code/game/machinery/kitchen/smartfridge.dm

```diff
@@ -27,15 +27,20 @@
 									/obj/item/weapon/reagent_containers/food/snacks/egg,
 									/obj/item/weapon/reagent_containers/food/condiment)
 
-	machine_flags = SCREWTOGGLE | CROWDESTROY | EJECTNOTDEL | WRENCHMOVE
+	machine_flags = SCREWTOGGLE | CROWDESTROY | EJECTNOTDEL | WRENCHMOVE | FIXED2WORK
 
 	light_color = LIGHT_COLOR_CYAN
-	power_change()
-		..()
-		if(!(stat & (BROKEN|NOPOWER)))
+
+/obj/machinery/smartfridge/power_change()
+	if( powered() )
+		stat &= ~NOPOWER
+		set_light(0)
+	else
+		spawn(rand(0, 15))
```

Comment:
why is this here and/or why is the rest of the code not indented under it?

Reply:
It is here because the OG coders copy-pasted the same code all over the codebase
It is not intended because I made a dumb ; will fix